net-ftp/frox/frox-0.7.18-r6.ebuild

```diff
@@ -45,6 +45,7 @@ src_prepare () {
 	eapply "${FILESDIR}/${PV}-respect-CFLAGS.patch"
 	eapply "${FILESDIR}/${PV}-netfilter-includes.patch"
 	eapply "${FILESDIR}/${P}-config.patch"
+	eapply "${FILESDIR}/${P}-no-common.patch"
```

Comment:
Please also bump the copyright year and author for this ebuild. You can do it manually by committing with repoman tool.

Reply:
I've just rebased and commited via repoman. The copyright year did not need to be changed, since it is already in upstream.